README.adoc

```diff
@@ -249,77 +317,83 @@ environment for you to write tests.
 
-In addition, a few endpoint tests `src/test/java/it/io/openliberty/guides/inventory/InventoryEndpointTest.java`
-and `src/test/java/it/io/openliberty/guides/system/SystemEndpointTest.java` are provided for you to
+In addition, a few endpoint tests `src/test/java/it/io/openliberty/guides/inventory/InventoryEndpointIT.java`
+and `src/test/java/it/io/openliberty/guides/system/SystemEndpointIT.java` are provided for you to
 test the basic functionality of the `inventory` and `system` services. If a test failure occurs, then you might have
 introduced a bug into the code.
 
 // =================================================================================================
 // Running the tests
 // =================================================================================================
-[role=command]
-include::{common-includes}/mvnverify.adoc[]
+
+=== Running the tests
+
+Since you started Open Liberty in development mode at the start of the guide, press the `enter/return` key to run the tests. 
+You will see the following output:
 
 [source, role="no_copy"]
 ----
 -------------------------------------------------------
  T E S T S
 -------------------------------------------------------
-Running it.io.openliberty.guides.system.SystemEndpointTest
-Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.4 sec - in it.io.openliberty.guides.system.SystemEndpointTest
-Running it.io.openliberty.guides.metrics.MetricsTest
-Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.697 sec - in it.io.openliberty.guides.metrics.MetricsTest
-Running it.io.openliberty.guides.inventory.InventoryEndpointTest
-Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.264 sec - in it.io.openliberty.guides.inventory.InventoryEndpointTest
+Running it.io.openliberty.guides.system.SystemEndpointIT
+Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.4 sec - in it.io.openliberty.guides.system.SystemEndpointIT
+Running it.io.openliberty.guides.metrics.MetricsIT
+Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.697 sec - in it.io.openliberty.guides.metrics.MetricsIT
+Running it.io.openliberty.guides.inventory.InventoryEndpointIT
+Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.264 sec - in it.io.openliberty.guides.inventory.InventoryIT
 
 Results :
 
 Tests run: 3, Failures: 0, Errors: 0, Skipped: 0
 ----
 
-To see whether the tests detect a failure, go to the [hotspot file=0]`MetricsTest.java` file and change any of the assertions
-in the test methods. Rerun the Maven build. A test failure occurs.
+To see whether the tests detect a failure, go to the [hotspot file=0]`MetricsIT.java` file and change any of the assertions
+in the test methods. Rerun the tests to see a test failure occur.
+
+When you are done checking out the service, exit development mode by typing `q` in the shell session 
```

Comment:
change
When you are done checking out the service, exit development mode by typing `q` in the shell session 
where you ran the server and then pressing the `enter/return` key.
TO
**When you have finished examining the service, exit out of development mode by typing `q` in the shell session 
where you ran the server and then press the `ENTER/RETURN` key.**